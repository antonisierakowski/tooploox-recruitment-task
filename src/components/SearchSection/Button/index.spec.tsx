import { render } from '@testing-library/react';
import React from 'react';
import { Button } from './index';

it('should match to the generated snapshot', () => {
  const { asFragment } = render(<Button caption="test" />);
  expect(asFragment()).toMatchSnapshot();
});
