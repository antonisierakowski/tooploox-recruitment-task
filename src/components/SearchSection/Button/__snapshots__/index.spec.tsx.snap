// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match to the generated snapshot 1`] = `
<DocumentFragment>
  <button
    class="button"
  >
    test
  </button>
</DocumentFragment>
`;
